lib/error.js

```diff
@@ -619,6 +621,24 @@ AppError.unverifiedPrimaryEmailNewlyCreated = () => {
   })
 }
 
+AppError.cannotLoginWithSecondaryEmail = () => {
+  return new AppError({
+    code: 400,
+    error: 'Bad Request',
+    errno: ERRNO.LOGIN_WITH_SECONDARY_EMAIL,
```

Comment:
We'll have to add these error numbers to the content server or else [we might end up with strange problems once these surface](https://github.com/mozilla/fxa-content-server/issues/4906).